### PR TITLE
Added textAppearance property for TextView

### PR DESCRIPTION
**Pull request by arslancharyev31**

### CODE REVIEW

File: anko/library/generated/sdk25/src/Properties.kt [hunk 1]
```diff
@@ -122,3 +122,12 @@ var android.widget.Toolbar.titleResource: Int
     @Deprecated(AnkoInternals.NO_GETTER, level = DeprecationLevel.ERROR) get() = AnkoInternals.noGetter()
     set(v) = setTitle(v)
 
+var android.widget.TextView.textAppearance: Int
+	@Deprecated(AnkoInternals.NO_GETTER, level = DeprecationLevel.ERROR) get() = AnkoInternals.noGetter()
```
Comment (piotrek1543) [inline anko/library/generated/sdk25/src/Properties.kt]: Could you explain why did you add this in `sdk25` folder? It already exists in `sdk23` version of `Properties` file.

**Comment by arslancharyev31** [comment]
Really? About which branch are you talking about? I've checked out master, and it doesn't seem to have one: https://github.com/Kotlin/anko/blob/master/anko/library/generated/sdk23/src/Properties.kt
Btw, what's the purpose of these packages ending with sdk number?

File: anko/library/generated/sdk25/src/Properties.kt (hunk 1 above)
Comment (arslancharyev31) [inline anko/library/generated/sdk25/src/Properties.kt]: Really? About which branch are you talking about? I've checked out master, and it doesn't seem to have one: https://github.com/Kotlin/anko/blob/master/anko/library/generated/sdk23/src/Properties.kt
Btw, what's the purpose of these packages ending with sdk number?

**Comment by piotrek1543** [comment]
I've founded it here: https://github.com/Kotlin/anko/blob/8f8a6c7485fab9c8492f2ccfd5cecb97e3ce7f30/anko/library/static/platform/src/CustomViewProperties.kt

but still I have no clue why you added to sdk25, not to sdk23/src/Properties.kt... You're checking if there is M or higher Android version, right?

**Comment by arslancharyev31** [comment]
Well yes, your are absolutely correct, I should've added it in sdk-23. Actually, I shouldn't have added it since it is already in master, it is just not released in new version yet.
